<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>323</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EXT75DX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Speed:</string>
      </property>
      <property name="buddy">
       <cstring>speed</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="speed">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> rps</string>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="vent_valve_state">
      <property name="text">
       <string>OPEN</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="8">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="start_pump">
          <property name="text">
           <string>Start Pump</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="stop_pump">
          <property name="text">
           <string>Stop Pump</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="close_vent_valve">
          <property name="text">
           <string>Close Vent Valve</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QRadioButton" name="standby">
          <property name="text">
           <string>Standby</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="reset_pump">
          <property name="text">
           <string>Reset Pump</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QRadioButton" name="normal">
          <property name="text">
           <string>Normal</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Pump Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Power Limit:</string>
          </property>
          <property name="buddy">
           <cstring>power_limit</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="power_limit">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> W</string>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Standby Speed:</string>
          </property>
          <property name="buddy">
           <cstring>standby_speed</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Venting Mode:</string>
          </property>
          <property name="buddy">
           <cstring>vent_mode</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="normal_speed">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Normal Speed:</string>
          </property>
          <property name="buddy">
           <cstring>normal_speed</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="standby_speed">
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="minimum">
           <number>55</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>70</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="vent_mode">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>Hard vent only when &lt;50% speed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Controlled vent if &gt;50% speed or
hard vent if &lt;50% speed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hard vent if stop or hard vent if fail
and &lt;50% speed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hard vent if stop or controlled vent
if fail and &gt;50% speed or hard vent if
fail and &lt;50% speed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hard vent if fail or hard vent if stop
and &lt;50% speed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hard vent if fail or controlled vent if
stop and &gt;50% speed or hard vent if
stop and &lt;50% speed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hard vent if stop or fail</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hard vent if stop or fail</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Permanently Enabled (Fan)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Temperature Module:</string>
      </property>
      <property name="buddy">
       <cstring>temp_module</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="temp_module">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> Â°C</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Power Consumption:</string>
      </property>
      <property name="buddy">
       <cstring>power_consumption</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Vent Valve:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="refresh">
      <property name="text">
       <string>Refresh Values</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="power_consumption">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> W</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="temp_motor">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> Â°C</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Temperature Motor:</string>
      </property>
      <property name="buddy">
       <cstring>temp_motor</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
